### ISSUE TITLE
How to test primaryAction on a Menu

### ISSUE THREAD

**Issue by travischapman** [nalexn/ViewInspector#183]
In Apple's [documentation for Menu](https://developer.apple.com/documentation/swiftui/menu), one of the initializers supports a `primaryAction`. When this initializer is used, the primary action is called when the Menu's label is tapped. If the user long presses the label, then the typical Menu is shown.

Example SwiftUI code:
```
Menu {
    Button(action: addCurrentTabToReadingList) {
        Label("Add to Reading List", systemImage: "eyeglasses")
    }
    Button(action: bookmarkAll) {
        Label("Add Bookmarks for All Tabs", systemImage: "book")
    }
    Button(action: show) {
        Label("Show All Bookmarks", systemImage: "books.vertical")
    }
} label: {
    Label("Add Bookmark", systemImage: "book")
} primaryAction: {
    addBookmark()
}
```

Is there support for testing the `primaryAction` of a Menu with ViewInspector?

**Comment by nalexn** [comment]
Thank you for the request, @travischapman ! v0.9.2 will have support for `primaryAction`

**Comment by nalexn** [comment]
Released with v0.9.2
